release/pkg/manager/calico/manager.go

```diff
@@ -891,7 +897,7 @@ func (r *CalicoManager) generateManifests() error {
 }
 
 func (r *CalicoManager) resetManifests() {
-	if _, err := r.runner.RunInDir(r.repoRoot, "git", []string{"checkout", "manifests"}, nil); err != nil {
+	if _, err := r.runner.RunInDir(r.repoRoot, "git", []string{"checkout", "manifests", "charts"}, nil); err != nil {
```

Comment:
What change happens in the charts diretory? I can't see where we're modifying that

Reply:
the images in `charts/calico/values.yaml` have their registies updated by `manifests/generate.sh` [here](https://github.com/projectcalico/calico/pull/9754/files#diff-3c90671bf6dbbd7436fde8f5512b677ebc4846700d80dc63635a30b4b58a68afR34)

Reply:
Ah, right. I need to think about that one for a moment. I think it's a change in scope of the `generate.sh` script that I might not be completely comfortable with. That script is meant to deterministically generate manifests based on values.yaml, not be responsible for modifying values.yaml itself.